src/main/c/sslcontext.c

```diff
@@ -1591,6 +1591,27 @@ TCN_IMPLEMENT_CALL(jboolean, SSLContext, setSessionIdContext)(TCN_STDARGS, jlong
     }
     return JNI_FALSE;
 }
+
+TCN_IMPLEMENT_CALL(jint, SSLContext, setMode)(TCN_STDARGS, jlong ctx, jint mode)
+{
+    tcn_ssl_ctxt_t *c = J2P(ctx, tcn_ssl_ctxt_t *);
+
+    UNREFERENCED(o);
```

Comment:
@normanmaurer +1. I was more concerned about if it was harmful or not ... but doesn't seem that way(?).